log4net.Azure/packages.config

```diff
@@ -1,11 +1,11 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="log4net" version="2.0.3" targetFramework="net45" />
+  <package id="log4net" version="2.0.8" targetFramework="net45" />
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
   <package id="Microsoft.Data.Edm" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.Data.OData" version="5.6.4" targetFramework="net45" />
   <package id="Microsoft.Data.Services.Client" version="5.6.4" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="10.0.2" targetFramework="net45" />
   <package id="System.Spatial" version="5.6.4" targetFramework="net45" />
-  <package id="WindowsAzure.Storage" version="8.7.0" targetFramework="net45" />
+  <package id="WindowsAzure.Storage" version="9.3.3" targetFramework="net45" />
```

Comment:
Do we need 9.3.3 or can we set 9.0.0? Setting 9.0.0 means that the consuming project can choose something lower than 9.3.3 if they need to.

Reply:
Good point, I am not sure actually but I will test it! :)

Reply:
I does not appear like we can set it to anything other than 9.3.3. but I might be mistaken since I am not well-versed in this specifically.

Reply:
Do you mean that Episerver requires 9.3.3?

Reply:
I don't know, when I set it to 9.0.0. everything seems to go bananas, but I will try to reset my environment and try again tomorrow :)

Reply:
I don't mean that you should try to force your Epi env to use 9.0.0. :smile: 

If we set 9.3.3 in this package then no one will be able to use this package unless they have 9.3.3 or later. By setting 9.0.0 in this package our projects can decide for themselves if they want to use 9.3.3, 9.2.X or 9.9.Y, etc.

So the question was mostly if there's any known issues in 9.0.0 that means that we can't say "use any version between 9.0.0 and 10.0.0".

Reply:
😄 I thought it was worth a shot to try it in 9.0.0.

Episerver needs it to be 9.3.3. though :(
https://nuget.episerver.com/package/?id=EPiServer.Azure

Reply:
Right, but since this package doesn't have a dependency on Episerver we don't have to care about that here. :smile: